addons/l10n_gcc_invoice/views/report_invoice.xml

```diff
@@ -0,0 +1,605 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="report_invoice" inherit_id="account.report_invoice">
+        <xpath expr='//t[@t-call="account.report_invoice_document"]' position="after">
+            <t t-if="o._get_name_invoice_report() == 'l10n_gcc_invoice.ar_invoice'"
+               t-call="l10n_gcc_invoice.ar_invoice" t-lang="lang"/>
+        </xpath>
+    </template>
+
+    <template id="report_invoice_with_payments" inherit_id="account.report_invoice_with_payments">
+        <xpath expr='//t[@t-call="account.report_invoice_document"]' position="after">
+            <t t-if="o._get_name_invoice_report() == 'l10n_gcc_invoice.ar_invoice'"
+               t-call="l10n_gcc_invoice.ar_invoice" t-lang="lang"/>
+        </xpath>
+    </template>
+
+    <template id="ar_invoice">
+        <t t-call="l10n_gcc_invoice.external_layout_standard">
+            <t t-if="not company">
+                <!-- Multicompany -->
+                <t t-if="company_id">
+                    <t t-set="company" t-value="company_id"/>
+                </t>
+                <t t-elif="o and 'company_id' in o and o.company_id.sudo()">
+                    <t t-set="company" t-value="o.company_id.sudo()"/>
+                </t>
+                <t t-else="else">
+                    <t t-set="company" t-value="res_company"/>
+                </t>
+            </t>
+
+            <t t-set="o_sec" t-value="o.with_context(lang='ar_001')"/>
+            <t t-set="o" t-value="o.with_context(lang='en_US')"/>
+            <t t-set="infocol" t-value="('col-md-2' if report_type == 'html' else 'col-2')"/>
```

Comment:
Not sure where this comes from.  The portal view might need to be improved anyways.  Has this been done on purpose?

Reply:
yes this has been done for the portal view. This has been done in other report in standard including the original invoice template account.report_invoice_document